docs.ps1

```diff
@@ -0,0 +1,53 @@
+param([Switch]$D = $false)
+
+# Check working directory is the root directory of the PDKGO repository
+if ((Test-Path -Path ".\docs\md") -eq $false)
+{
+    Write-Host 'Please run this script from the root directory of the PDKGO project.'
+    Exit 1
+}
+# Check hugo extended is installed
+if ((hugo version |Select-String -Pattern "extended") -eq $null)
```

Comment:
```suggestion
if ([string]::IsNullOrEmpty(hugo version | Select-String -Pattern "extended"))
```

Minor style nit: always include a space before and after a pipe on the same line.

Moving to the more idiomatic `[string]::IsNullOrEmpty()` check also ensures both faster execution as well as handling for the weird edge case where `Select-String` could return an empty string, which would not equal `$null`.